### PR TITLE
New Segment: disk_usage

### PR DESCRIPTION
**Pull request by msempere**
Following PR:
 - Adds a new segment to display a specific filesystem usage
 - Allows customizing the desired filesystem ( `/` as default)

Output example using `/` filesystem:

screen shot 2018-03-25 at 20 11 39

### CODE REVIEW

File: segments/disk_usage.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	read -d '' rccontents  << EORC
+# Filesystem to retrieve disk space information. Any from the filesystems available (run "df | awk '{print $1}'" to check them).
+export TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM="${TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM_DEFAULT}"
+EORC
+	echo "$rccontents"
+}
+
+run_segment() {
+	__process_settings
+    percentage=`df ${TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM} | awk '{print $5}' | tail -n1`
```
Comment (erikw) [inline segments/disk_usage.sh]: Using $() is nice than ``, as it's visually easier to match a left parenthesis to a right one, and it's way clearer if you nest things

Comment (msempere) [reply segments/disk_usage.sh]: oh, I know what you mean.
Will address this. Thanks for the recommendation.

**Comment by msempere** [comment]
Done, please @erikw take another look.

**Comment by erikw** [comment]
Looks better, feel free to hit the merge button yourself :)

**Pull request merged**